### PR TITLE
backend/oss: Ignore the getting oss endpoint error and using string concat instead; Improves the error message level

### PR DESCRIPTION
**Pull request by xiaozhu36**
This PR aims to fix getting oss endpoint error caused by network connection failed and other reasons. This error will break oss backend process. Considering all of oss endpoints have followed up pattern `oss-<RegionId>.aliyuncs.com`, above error can be ignored.

Fixes #

## Target Release

1.4.x

###  ENHANCEMENTS 

 

-  backend/oss: Ignore the getting oss endpoint error and using string concat instead; Improves the error message level

### CODE REVIEW

**Comment by crw** [comment]
Thanks for this submission, I'll notify the core team.

File: internal/backend/remote-state/oss/backend.go
Comment (alisdair) [inline internal/backend/remote-state/oss/backend.go:383]: I think this is the line that `gofmt` is failing with.

**Review by alisdair — CHANGES_REQUESTED** [review]
Happy to merge this if you can fix the `gofmt` error! Please run `gofmt` on these files and push a new commit.

File: internal/backend/remote-state/oss/backend.go
Comment (xiaozhu36) [reply internal/backend/remote-state/oss/backend.go]: Thanks for your review and feedback and I have fix it. Please have a check again.

**Pull request merged**
